src/routes/api/index.js

```diff
@@ -1,8 +1,9 @@
 import express from 'express';
 import passport from 'passport';
+import userRoutes from './users';
 
 const usersRouter = express.Router();
-
+usersRouter.use('/', userRoutes);
 usersRouter.post('/users/login', (req, res, next) => {
   if (!req.body.user.email) {
     return res.status(422).json({ errors: { email: "can't be blank" } });
```

Comment:
@sebaztyn Let's stick to the status code of `400` here. I know there's a lot of debate on how and where to use `422` instead but we should stick to `400`. Also, the error message should be more like `email can't be blank` and not `can't be blank` so users can easily understand what entity the message is referring

Reply:
Dear @Stylll, I have made change changes. Thank you.